cmd/acra-keys/keys/migrate-keys.go

```diff
@@ -226,7 +226,7 @@ func MigrateV1toV2(srcV1 filesystem.KeyExport, dstV2 keystoreV2.KeyFileImportV1)
 
 	log.Tracef("Importing %d keys from keystore v1", expected)
 	for _, key := range keys {
-		log := log.WithField("purpose", key.Purpose).WithField("id", key.ID)
+		log := log.WithField("purpose", key.KeyContext.Purpose).WithField("id", keystore.GetKeyContextFromContext(key.KeyContext))
```

Comment:
But we are logging `key.KeyContext.Purpose` here or what do you mean?

Reply:
`.WithField("id", keystore.GetKeyContextFromContext(key.KeyContext))`
How it will look in log entry? Something like `&{ClientID: nil, ZoneID: nil, Context: []byte{0, 12, 42, 321}}` or something else?

Reply:
Not precisely, previously we had

```
log := log.WithField("purpose", key.Purpose).WithField("id", key.ID) 
```
where `key.ID` is simple bytes slice which is strange as it would print something like `[]byte{0, 12, 42, 321}`

So I just repeat the functionality. Here `key.KeyContext` is not bytes slice but `keystore.KeyContext` struct. So now it will print some bytes slice because of the function.
```
func GetKeyContextFromContext(keyContext KeyContext) []byte {
	if keyContext.ZoneID != nil {
		return keyContext.ZoneID
	}
	if keyContext.ClientID != nil {
		return keyContext.ClientID
	}
	if keyContext.Context != nil {
		return keyContext.Context
	}
	return nil
}
```
I think it makes sense to wrap with `string` to print string data instead of bytes @Lagovas

Reply:
+1